tests/bluetooth/bsim/nrf_auraconfig/tester/include/nac_test.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef _NAC_TEST_
+#define _NAC_TEST_
+
+#include <zephyr/bluetooth/audio/cap.h>
```

Comment:
```suggestion
#define _NAC_TEST_

/**
 * @defgroup auraconfig_tester nRF Auraconfig tester
 * @brief nRF Auraconfig tester
 *
 * @{
 */

#include <zephyr/bluetooth/audio/cap.h>
```
Do we need doxygen documentation for this file?

Reply:
Up to you. As mentioned this is just for testing internally in CI